src/CommunityToolkit/Xamarin.CommunityToolkit/Behaviors/Animations/AnimationTypes/ScaleAnimation.shared.cs

```diff
@@ -22,4 +22,22 @@ public override async Task Animate(View view)
 			await view.ScaleTo(1, Duration, Easing);
 		}
 	}
+
+	public class RelScaleAnimation : AnimationBase
+	{
+		public static readonly BindableProperty ScaleProperty =
+			BindableProperty.Create(nameof(Scale), typeof(double), typeof(RelScaleAnimation), default(double),
+				BindingMode.TwoWay, null);
+
+		public double Scale
+		{
+			get => (double)GetValue(ScaleProperty);
+			set => SetValue(ScaleProperty, value);
+		}
+
+		protected override uint DefaultDuration { get; set; } = 170;
+
+		public override async Task Animate(View view) =>
+			await view.RelScaleTo(Scale, Duration, Easing);
```

Comment:
All of the other `override Animate` methods push to the main thread. Should this push to the main thread too?

```csharp
public override Task Animate(View view) => Device.InvokeOnMainThreadAsync(() => view.RelScaleTo(Scale, Duration, Easing));
```